### ISSUE TITLE
Any updates? There has been a bunch of new errors lately. 

### ISSUE THREAD

**Issue by koderian** [LjMario007/Microsoft-Rewards-Bot#53]
While the bot is still working, to some extent, there have been a bunch of errors slowly adding up. I just think a update from LjMario007 would be good at this point. #37 #49 #32 #51 are all issues at the moment.

Ubuntu 20.04

Gracias

**Comment by kowshika-n** [comment]
#54 resolves some of the issues.

**Comment by coollbv** [comment]
Not really... Getting a lot of errors still and no searches are recorded...

**Comment by LjMario007** [comment]
Apologies, I haven't quite gotten the chance to okay very many PRs lately. Can I add y'all as contributors, @kowshika-n or @koderian?

**Comment by kowshika-n** [comment]
Sure. Thanks for considering me.

**Comment by ZeroCool940711** [comment]
Is there any news on fixing the errors?
Im getting a lot of errors, even more after the latest PR was accepted, PC, Mobile and Edge Browser searches are not, even some daily quizzes are failing, looking at the logs is more errors what I see than anything else, also the bot thinks it finished searches correctly but in reality it hasn't made a single search successfully.
[ms_rewards.log](https://github.com/LjMario007/Microsoft-Rewards-Bot/files/5382958/ms_rewards.log)

**Comment by velotoxicsounds** [comment]
Watch Here @ZeroCool940711  -> https://github.com/LjMario007/Microsoft-Rewards-Bot/pull/64
